elliottlib/cli/find_bugs_cli.py

```diff
@@ -130,6 +130,9 @@ def find_bugs_cli(runtime, advisory, default_advisory_type, mode, check_builds,
     if mode == 'list' and len(id) == 0:
         raise click.BadParameter("When using mode=list, you must provide a list of bug IDs")
 
+    if mode == 'list' and into_default_advisories:
```

Comment:
Although this is rare but I think `list` mode can be used with `--into-default-advisories`. `list` just means "use specified bugs"

Reply:
I think if not necessary we shouldn't support such a rare usecase (of attaching same bugs to multiple advisories). I checked with Luke, and we agreed on not having this option usable. Let me know if you still think we should have it